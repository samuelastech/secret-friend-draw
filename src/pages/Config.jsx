import { Form, Footer, Header, ParticipantsList } from '../components/index';

export const Config = () => {
  return (
    <>
      <Header />
      <Form />
      <ParticipantsList />
      <Footer />
    </>
  );
};
