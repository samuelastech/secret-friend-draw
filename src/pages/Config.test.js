import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import { Config } from "./Config";

jest.mock('react-router-dom');

test('should render the page', () => {
  const { container } = render(
    <RecoilRoot>
      <Config />
    </RecoilRoot>
  );

  expect(container).toMatchSnapshot();
});