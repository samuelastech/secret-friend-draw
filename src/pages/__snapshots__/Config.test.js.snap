// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the page 1`] = `
<div>
  <div>
    <header>
      <img
        alt="Secret friend drawer's logo"
        src="/images/logo.png"
      />
      <img
        alt="Avatar image of a participant"
        src="/images/person.png"
      />
    </header>
  </div>
  <form>
    <input
      placeholder="Insira o nome do partcipante"
      type="text"
      value=""
    />
    <button
      disabled=""
      type="submit"
    >
      Adicionar
    </button>
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Come√ßar a brincadeira
    </button>
  </footer>
</div>
`;
